pkg/operator/ceph/file/filesystem.go

```diff
@@ -72,6 +73,11 @@ func createFilesystem(
 		}
 	}
 
+	err := cephclient.CreateCephFSSubVolumeGroup(context, clusterInfo, fs.Name, defaultCSISubvolumeGroup)
```

Comment:
I think we should create the CR, instead of directly calling it, 
As we need to reconcile it + we need to call pinning + we have use new name spec if multiple filesystem is created(currently it also handled with your change).

Reply:
I believe this was already resolved in the separate comments, but just to clarify... We will create svg here to ensure it exists, and the CR for the default svg will really just be needed for pinning settings, which would then be applied by that svg controller. Correct?